database_backup/02-cronjob.yaml

```diff
@@ -10,6 +10,7 @@ spec:
   failedJobsHistoryLimit: 3
```

Comment:
this job is run 1st day of every, so setting the job to be cleaned after 4 days of running.